go.mod

```diff
@@ -5,8 +5,18 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/jhump/protoreflect v1.12.0
 	github.com/stretchr/testify v1.8.0
-	golang.org/x/net v0.0.0-20201021035429-f5854403a974
-	google.golang.org/grpc v1.47.0
+	golang.org/x/net v0.0.0-20220708220712-1185a9018129
+	google.golang.org/grpc v1.48.0
 )
 
-go 1.16
+require (
```

Comment:
Out of curiosity, was this added after running `go mod tidy`?

Reply:
It's a go 1.17 thing

https://golang.org/doc/go1.17#go-command

"... If a module specifies go 1.17 or higher in its go.mod file, its go.mod file now contains an explicit require directive for every module that provides a transitively-imported package. (In previous versions, the go.mod file typically only included explicit requirements for directly-imported packages.)"

Reply:
Cool 👍